action.yaml

```diff
@@ -34,9 +34,16 @@ runs:
         project_name=${repository:$repository_characters_to_skip}
 
         # Determine the target path
-        short_sha=${sha:0:8} # First 8 characters of the commit sha
+        short_sha=$(git rev-parse --short HEAD)
         test_result_target_path="s3://${REGULATORY_S3_BUCKET}/projects/${project_name}/commits/${short_sha}/test-results.xml"
-        
+
+        date=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+        run_number=${{ github.run_number }}
+        name="Docker_Image"
```

Comment:
(very, very minor) Any specific reason this doesn't have a space? As I think the SLT did create a folder with a space. Personal preference, if not might be nicer to go for `Docker-Image`

Reply:
The space did not work in the github action, I can take a look if I can escape it somehow, otherwise I will change it to a dash which is indeed nicer.